src/test/java/uk/gov/digital/ho/hocs/casework/domain/model/CaseDataTest.java

```diff
@@ -32,7 +32,6 @@ public void getCaseData() {
         assertThat(caseData.getPrimaryTopicUUID()).isNull();
         assertThat(caseData.getCaseDeadline()).isNull();
         assertThat(caseData.getDateReceived()).isEqualTo(caseReceived);
-        assertThat(caseData.isCompleted()).isEqualTo(false);
```

Comment:
This and the other modified tests in this file should be replaced with a check that date completed has been set.